core/coreapi/interface/interface.go

```diff
@@ -17,9 +17,13 @@ import (
 // Path is a generic wrapper for paths used in the API. A path can be resolved
 // to a CID using one of Resolve functions in the API.
 type Path interface {
+	// String returns the path as a string.
 	String() string
+	// Cid returns path cid
 	Cid() *cid.Cid
+	// Root returns path root
 	Root() *cid.Cid
+	// Resolved returns whether cid is nil
```

Comment:
While this may be technically true, I think we want to say whether or not this path has been fully resolved.

cc @magik6k @lgierth